### PR TITLE
Enrollment validation for first name.

### PR DESCRIPTION
**Pull request by nyb12**

### CODE REVIEW

File: apps/front-end/src/pages/front-end/Attendence/Attendence.js [hunk 1]
```diff
@@ -274,11 +274,11 @@ export default function Attendence({ footerLinks }) {
...
           bg={
-            row?.user?.aadhar_verified !== "yes"
-              ? "dangerColor"
-              : "potentialColor"
+            row?.user?.aadhar_verified !== null
+              ? "potentialColor"
+              : "dangerColor"
           }
-          label={row?.user?.aadhar_verified !== "yes" ? t("NO") : t("YES")}
+          label={row?.user?.aadhar_verified !== null ? t("YES") : t("NO")}
```
Comment (sagarkoshti1990) [inline apps/front-end/src/pages/front-end/Attendence/Attendence.js:281]: not showing `in_progress`??

File: apps/front-end/src/pages/front-end/Attendence/Attendence.js [hunk 2]
```diff
@@ -890,16 +884,25 @@ export default function Attendence({ footerLinks }) {
...
+                                    navigate(
+                                      `/aadhaar-kyc/${formData?.user_id}`,
+                                      {
+                                        state: {
+                                          returnUrl: `/attendence/${formData?.context_id}`,
+                                        },
+                                      }
+                                    );
+                                  }}
+                                /> */}
```
Comment (sagarkoshti1990) [inline apps/front-end/src/pages/front-end/Attendence/Attendence.js:904]: Remove if not needed.

File: apps/front-end/src/pages/front-end/ag-edit/choose-subjects/ErollmentReceipt.js [hunk 3]
```diff
@@ -134,15 +134,17 @@ export default function App({ facilitator }) {
...
       enrollment_middle_name: enrollment_middle_name
         ? enrollment_middle_name
         : "",
       enrollment_last_name: enrollment_last_name ? enrollment_last_name : "",
-      enrollment_dob: enrollment_dob ? enrollment_dob : "",
+      enrollment_dob: enrollment_dob ? enrollment_dob : undefined,
       enrollment_aadhaar_no: enrollment_aadhaar_no
         ? parseInt(enrollment_aadhaar_no)
-        : "",
+        : null,
```
Comment (sagarkoshti1990) [inline apps/front-end/src/pages/front-end/ag-edit/choose-subjects/ErollmentReceipt.js:147]: change `null` to `undefined`

File: apps/front-end/src/pages/front-end/ag-edit/choose-subjects/subjectForm.js [hunk 4]
```diff
@@ -136,10 +138,7 @@ export default function App({ facilitator, id, ip, onClick }) {
       "ui:options": {
         hideNowButton: true,
         hideClearButton: true,
-        yearsRange: [
-          moment().subtract("years", 30).format("YYYY"),
-          moment().format("YYYY"),
-        ],
+        yearsRange: [2023, moment().format("YYYY")],
```
Comment (sagarkoshti1990) [inline apps/front-end/src/pages/front-end/ag-edit/choose-subjects/subjectForm.js:141]: why remove 30 yeas condition ?

**Review by sagarkoshti1990 — CHANGES_REQUESTED** [review]

File: apps/front-end/src/pages/front-end/Attendence/Attendence.js (hunk 2 above)
Comment (nyb12) [reply apps/front-end/src/pages/front-end/Attendence/Attendence.js:904]: its needed but its not navigating back to attendence form after complete aadhaar kyc so i commented this code

File: apps/front-end/src/pages/front-end/ag-edit/choose-subjects/subjectForm.js (hunk 4 above)
Comment (nyb12) [reply apps/front-end/src/pages/front-end/ag-edit/choose-subjects/subjectForm.js:141]: it was requested by sir to keep it from there year

File: apps/front-end/src/pages/front-end/Attendence/Attendence.js (hunk 1 above)
Comment (nyb12) [reply apps/front-end/src/pages/front-end/Attendence/Attendence.js:281]: showing yes only there

**Review by sagarkoshti1990 — APPROVED** [review]

**Pull request merged**